modules/keypad.h

```diff
@@ -32,6 +32,9 @@
 #define MCE_KEYBOARD_BACKLIGHT_BRIGHTNESS_SYS_PATH	MCE_KEYBOARD_BACKLIGHT_SYS_PATH "/brightness"
 #define MCE_KEYBOARD_BACKLIGHT_FADETIME_SYS_PATH	MCE_KEYBOARD_BACKLIGHT_SYS_PATH "/time"
 
+#define MCE_KEYBOARD_XT894_BACKLIGHT_SYS_PATH	"/sys/class/leds/lm3532::kbd_backlight/brightness"
```

Comment:
Style inconsistency. It's better to align the define's value like it's done in above lines.